example/src/main/scala/com/avast/server/toolkit/example/Main.scala

```diff
@@ -7,14 +7,16 @@ import com.avast.server.toolkit.example.config.Configuration
 import com.avast.server.toolkit.execution.ExecutorModule
 import com.avast.server.toolkit.pureconfig.PureConfigModule
 import com.avast.server.toolkit.system.console.{Console, ConsoleModule}
+import com.github.ghik.silencer.silent
 import zio.interop.catz._
 import zio.{Task, ZIO}
 
 object Main extends CatsApp {
 
+  @SuppressWarnings(Array("org.wartremover.warts.StringPlusAny")) // false positive
```

Comment:
I would prefer to disable this wart completely as it can produce false positive.

Reply:
@sideeffffect Are you kidding me? I haven't noticed this but tell me where in this piece of code I am doing anything with strings and `+`? That's ridiculous, the linters do not work well at all.

Reply:
as the comment suggests, it's a false possitive
I think we can tolerate one occurence, because it could save us a lot of trouble